test/unit/operations/get_more.test.ts

```diff
@@ -87,12 +87,10 @@ describe('GetMoreOperation', function () {
         const session = sinon.createStubInstance(ClientSession);
         const opts = { ...options, session };
         const operation = new GetMoreOperation(namespace, cursorId, server1, opts);
-        const callback = error => {
+        await operation.execute(server2, session).catch(error => {
           expect(error).to.be.instanceOf(MongoRuntimeError);
           expect(error.message).to.equal('Getmore must run on the same server operation began on');
-          done();
-        };
-        operation.executeCallback(server2, session, callback);
+        });
```

Comment:
This is totally correct code, there's potentially just a nicer way to format it. See the patterns we've used for error assertions elsewhere ([example](https://github.com/mongodb/node-mongodb-native/blob/NODE-5392/test/integration/node-specific/mongo_client.test.ts#L610)). You can make the catch instead return the error, and assert on the result.

Reply:
Got it, done